int64 stepper_id
int64 steps
---
bool result
int64 encoder